debezium-core/src/main/java/io/debezium/schema/DatabaseSchema.java

```diff
@@ -15,6 +15,8 @@
  */
 public interface DatabaseSchema<I extends DataCollectionId> {
 
+    String NO_CAPTURED_DATA_COLLECTIONS_WARNING = "After applying include/exclude list filters there are no tables to monitor, please check your configuration";
```

Comment:
For MongoDB we shouldn't talk about tables. Perhaps "table/collections"? and please replace "monitor" with "to capture changes from". We want to get away from "monitor" in this context, and keep that for, well, monitoring the connectors via JMX etc.

Reply:
It's the same method as before. will log a ticket for that.

Reply:
will do `String NO_CAPTURED_DATA_COLLECTIONS_WARNING = "After applying include/exclude list filters there's nothing selected to capture changes from, please check your configuration";`

Reply:
Done